packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts

```diff
@@ -12,6 +12,8 @@ export type HintDisplayText = {
   hintIlluminationNormalText?: string;
   hintHoldFaceForFreshnessText?: string;
   hintCenterFaceText?: string;
+  hintFaceOffCenterText?: string;
+  hintMatchIndicatorA11yText?: string;
```

Comment:
`A11y` is not always a known acronym, think we can just remove it here:

```suggestion
  hintMatchIndicatorText?: string;
```